types/node-resque/index.d.ts

```diff
@@ -36,14 +36,46 @@ export interface QueueOptions {
     connection?: ConnectionOptions;
 }
 
+type WorkerStatus = {
```

Comment:
I think using interfaces over types is still preferred in that repository?
BTW That's one of the reasons why the build fails.